code/modules/power/singularity/singularity.dm

```diff
@@ -298,7 +298,7 @@
 
 
 /obj/singularity/proc/move(force_move = 0)
-	if(!move_self)
+	if(!move_self && !force_move)
```

Comment:
Non force move before made it move randomly, why are you removing that capability?